std_msgs/Float32MultiArray stateVec
---
int64 direction
